buildozer/targets/android.py

```diff
@@ -279,6 +279,10 @@ def check_requirements(self):
                     'zlib headers must be installed, '
                     'run: sudo apt-get install zlib1g-dev')
 
+            # Set environment to use Linux binaries on FreeBSD at later stages.
```

Comment:
```suggestion
            # Overrides the OS which `sdkmanager` should download the packages for.
            # See: https://developer.android.com/studio/command-line/variables#envar
```

Reply:
THanks @misl6 I have updated the description with your hints and why this is used :-)

Reply:
Sorry, i amended that to 64-bit detection commit, fix incoming :-)

Reply:
rebased :-)